### ISSUE TITLE
"View Theme Source" link on LinkOverlay is a 404

### ISSUE THREAD

**Issue by andrewcohen** [chakra-ui/chakra-ui-docs#473]
### Subject

LinkOverlay

### Description

"View Theme Source" on [this page](https://chakra-ui.com/docs/components/navigation/link-overlay) takes you to a [404](https://github.com/chakra-ui/chakra-ui/tree/main/packages/theme/src/components/link-overlay.ts).

From what I can tell from the component source, `LinkOverlay` and `LinkBox` are not actually able to be customized using the theme, but i'm not sure if that is intentional.

**Comment by nikolovlazar** [comment]
You're right, @andrewcohen! I'll push an update that fixes this. Thank you for raising this issue.
